### DISCUSSION TITLE
Temperature Sensor

### DISCUSSION THREAD

**Discussion by DTxAg** [potmat/homebridge-google-nest-sdm#82] [Q&A]
Is there a way for this plugin to pull in a Nest temperature sensor? I have one in a different room that a Nest thermostat, but I can't seem to get Homebridge to find it. When I add the Nest bridge to Apple Home, it just pulls in the thermostat and not the temperature sensor.

**Comment by potmat** [comment]
Unfortunately the SDM API does not support temperature sensors: https://developers.google.com/nest/device-access/supported-devices
